Stack/Opc.Ua.Core/Types/BuiltIn/ExtensionObject.cs

```diff
@@ -406,12 +399,10 @@ public ExtensionObjectEncoding Encoding
         /// 	</list>
         /// </remarks>
         /// <exception cref="ServiceResultException">Thrown when the body is not one of the types listed above</exception>
-        public object Body
-        {
+        public object Body {
```

Comment:
I see the curly bracket is put on the same line for properties and for _get_ and _set_ blocks in multiple places.
That is different than the actual code style.
The cause is in ._editorconfig_ file. and PR #899 should fix it.

Reply:
Thanks for the PR I will fix the formatting...